src/index.html

```diff
@@ -5,8 +5,339 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <div class="header__content container">
+          <div class="logo">
+            <a href="#home" class="logo__link">
+              <img
+                src="../src/images/logo.png"
+                alt="logo"
+                class="logo__img"
+              >
+            </a>
+          </div>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
+              <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+              <li class="nav__item"><a href="#smarthphones" class="nav__link">Smartphones</a></li>
+              <li class="nav__item">
+                <a
+                  data-qa="nav-hover"
+                  href="#laptopscomputers"
+                  class="nav__link"
+                >
+                  Laptops &#38; computers
+                </a>
+              </li>
+              <li class="nav__item"><a href="#gadgets" class="nav__link">Gadgets</a></li>
+              <li class="nav__item"><a href="#tablets" class="nav__link">Tablets</a></li>
+              <li class="nav__item"><a href="#photo" class="nav__link">Photo</a></li>
+              <li class="nav__item"><a href="#video" class="nav__link">Video</a></li>
```

Comment:
<img width="1159" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/de64ef84-703b-4d80-9529-1636447936ab">
Fix this in all code